src/textures/TextTextureRenderer.mjs

```diff
@@ -293,7 +294,17 @@ export default class TextTextureRenderer {
         this._context.fillStyle = StageUtils.getRgbaString(this._settings.textColor);
         for (let i = 0, n = drawLines.length; i < n; i++) {
             let drawLine = drawLines[i];
-            this._context.fillText(drawLine.text, drawLine.x, drawLine.y);
+
+            if (!letterSpacing) {
+                this._context.fillText(drawLine.text, drawLine.x, drawLine.y);
+            } else {
+                const textSplit = drawLine.text.split('');
+                let x = drawLine.x;
+                for (let i = 0, j = textSplit.length; i < j; i++) {
+                    this._context.fillText(textSplit[i], x, drawLine.y);
+                    x += this._context.measureText(textSplit[i]).width + letterSpacing;
```

Comment:
You can use `measureText` wrapper you created here.

Reply:
updated